app/api/ada/lib/storage/database/prepackaged/explorers.js

```diff
@@ -7,51 +7,57 @@ const CardanoMainnetExplorers: Array<$ReadOnly<ExplorerRow>> = [
   {
     ExplorerId: 1_00,
     NetworkId: networks.CardanoMainnet.NetworkId,
-    IsBackup: false,
+    IsBackup: true,
     Endpoints: {
-      address: 'https://adaex.org/',
-      transaction: 'https://adaex.org/',
-      pool: 'https://adapools.org/pool/',
+      address: 'https://cardanoscan.io/address/',
+      transaction: 'https://cardanoscan.io/transaction/',
+      pool: 'https://cardanoscan.io/pool/',
+      stakeAddress: 'https://cardanoscan.io/stakeKey/',
+      token: 'https://cardanoscan.io/token/',
     },
-    Name: 'ADAex.org',
+    Name: 'CardanoScan',
   },
   {
     ExplorerId: 1_01,
     NetworkId: networks.CardanoMainnet.NetworkId,
     IsBackup: false,
     Endpoints: {
-      address: 'https://adascan.net/address/',
-      transaction: 'https://adascan.net/transaction/',
+      address: 'https://adastat.net/addresses/',
+      transaction: 'https://adastat.net/transactions/',
+      pool: 'https://adastat.net/pools/',
     },
-    Name: 'AdaScan',
+    Name: 'AdaStat',
   },
   {
     ExplorerId: 1_02,
     NetworkId: networks.CardanoMainnet.NetworkId,
     IsBackup: false,
     Endpoints: {
-      address: 'https://blockchair.com/cardano/address/',
-      transaction: 'https://blockchair.com/cardano/transaction/',
+      address: 'https://explorer.cardano.org/en/address?address=',
+      transaction: 'https://explorer.cardano.org/en/transaction?id=',
     },
-    Name: 'Blockchair',
+    Name: 'CardanoExplorer',
   },
   {
     ExplorerId: 1_03,
     NetworkId: networks.CardanoMainnet.NetworkId,
     IsBackup: false,
     Endpoints: {
+      address: 'https://adaex.org/',
+      transaction: 'https://adaex.org/',
+      pool: 'https://adapools.org/pool/',
     },
-    Name: 'Clio.1',
+    Name: 'ADAex.org',
   },
   {
     ExplorerId: 1_04,
     NetworkId: networks.CardanoMainnet.NetworkId,
     IsBackup: false,
     Endpoints: {
-      address: 'https://explorer.cardano.org/en/address?address=',
```

Comment:
Please don't  change it so that ExplorerIds now point to different explorers 🙏 
Better to just leave gaps and add to the list if needed.